pkg/rancher-desktop/assets/lima-config.yaml

```diff
@@ -128,10 +128,12 @@ provision:
     # openresty is backgrounding itself (and writes its own pid file)
     sed -i 's/^command_background/#command_background/' /etc/init.d/openresty
 
+    # configure proxy only when image-allow-list exists
```

Comment:
At this point, it's probably clearer to write our own init script, similar to what we do for k3s:
https://github.com/rancher-sandbox/rancher-desktop/blob/f5def92e49f4f6149b12935326bc22ae44020bee/pkg/rancher-desktop/assets/scripts/service-k3s.initd#L62-L66

Reply:
I don't know; you want to inline/overwrite both `/etc/init.d/containerd` and `/etc/init.d/docker` just because we add a single line to them?

I feel that replacing the init scripts will just create potential maintenance issues down the road because we won't track upstream changes from Alpine.

We may want eventually to provide our own openresty init script, but that is a separate issue.